sklearn/cluster/optics_.py

```diff
@@ -375,46 +369,53 @@ def fit(self, X, y=None):
         self.n_clusters_ = np.max(self.labels_)
         return self
 
-    # OPTICS helper functions; these should not be public #
-
-    def _expand_cluster_order(self, point, X, nbrs):
-        # As above, not parallelizable. Parallelizing would allow items in
-        # the 'unprocessed' list to switch to 'processed'
-        if self.core_distances_[point] <= self.max_eps:
-            while not self._processed[point]:
-                self._processed[point] = True
-                self.ordering_.append(point)
-                point = self._set_reach_dist(point, X, nbrs)
-        else:  # For very noisy points
-            self.ordering_.append(point)
-            self._processed[point] = True
-
-    def _set_reach_dist(self, point_index, X, nbrs):
-        P = np.array(X[point_index]).reshape(1, -1)
+    # OPTICS helper functions
+
+    def _expand_cluster_order(self, X, nbrs):
```

Comment:
For somebody who reads the original paper and comes to read the implementation, the function name may be a bit confusing. According to the paper, this function (with this implementation) is called `OPTICS`.